### PR TITLE
Prepare Status of the Document section for Candidate Recommendation.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/609.html" title="Last updated on Feb 12, 2021, 2:42 PM UTC (93dc906)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/609/6645c05...93dc906.html" title="Last updated on Feb 12, 2021, 2:42 PM UTC (93dc906)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,21 +193,34 @@
...
+    </p>
+
+    <p>
+To exit the W3C Candidate Recommendation phase, the W3C DID Working Group will
+require two things: 1) for normative statements that are machine testable, at
+least two interoperable implementations per feature, where each feature is
+defined as one or more normative statements in the specification, and 2) for
+normative statements that are only human-tesable, at least two demonstrations of
+implementation per feature, where each feature is defined as one or more
+normative statements in the specification.
```
Comment (brentzundel) [inline index.html:209]:
```suggestion
To exit the W3C Candidate Recommendation phase, the W3C DID Working Group will
require two things: 1) for normative statements that are machine testable, at least two interoperable implementations per feature, and 2) for normative statements that are
not machine testable, at least two demonstrations of implementation per feature. A
feature is defined as one or more normative statements in the specification.
```

**Review by brentzundel — COMMENTED** [review]
minor re-wording suggestion in an attempt to increase clarity.

File: index.html [hunk 2]
```diff
@@ -223,22 +234,28 @@
 Portions of the work on this specification have been funded by the United States
 Department of Homeland Security's (US DHS) Science and Technology Directorate
 under contracts HSHQDC-16-R00012-H-SB2016-1-002, and HSHQDC-17-C-00019, as well
-as  the US DHS Silicon Valley Innovation Program under contracts
-70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this specification does not
-necessarily reflect the position or the policy of the U.S. Government and no
-official endorsement should be inferred.
+as the US DHS Silicon Valley Innovation Program under contracts
+70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this
```
Comment (kdenhartog) [inline index.html:238]:
```suggestion
70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030, 70RSAT20T00000045. The content of this
```

**Review by kdenhartog — APPROVED** [review]
Just adding our contract number in there as well. Thanks for following up on it @msporny !

File: index.html [hunk 3]
```diff
@@ -223,22 +234,28 @@
...
-Robles, Brian Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu
-Sporny, Drummond Reed, Joe Andrieu, and Heather Vescent.
+Work on this specification has also been supported by the  <a
+href="https://www.weboftrust.info/">Rebooting the Web of Trust</a> community
+facilitated by Christopher Allen, Shannon Appelcline, Kiara Robles, Brian
+Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu Sporny, Drummond
+Reed, Joe Andrieu, and Heather Vescent. Development of this specification has
+also been supported by the <a href="https://w3c-ccg.github.io/">W3C Credentials
+Community Group</a>, which has been Chaired by Kim Hamilton Duffy, Joe Andrieu,
+Christopher Allen, Heather Vescent, and Wayne Chang.
```
Comment (iherman) [inline index.html:258]: I guess in every case like that, and to avoid unnecessary discussions, it is wise to put the names in alphabetical order...

**Review by iherman — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (OR13) [inline index.html:238]:
```suggestion
70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030, 70RSAT20T00000003, 70RSAT20T00000033. The content of this
```

Comment (msporny) [reply index.html:238]: Github doesn't let me apply two changes to the same line. I've merged your numbers with @kdenhartog's numbers.

File: index.html [hunk 4]
```diff
@@ -193,21 +193,32 @@
...
+Method specification authors provide experimental implementations designed to
+test the implementability of all of the features in this document.
+    </p>
+
+    <p>
+To exit the W3C Candidate Recommendation phase, the W3C DID Working Group will
+require two things: 1) for normative statements that are machine testable, at least two
+interoperable implementations per feature, and 2) for normative statements that are
+not machine testable, at least two demonstrations of implementation per feature. A
+feature is defined as one or more normative statements in the specification.
```
Comment (msporny) [inline index.html:207]:
```suggestion
feature is defined as one or more related normative statements in the specification.
```

File: index.html [hunk 5]
```diff
@@ -193,21 +193,32 @@
...
     </p>
```
[the remark this answers is not in the document]
Comment (msporny) [reply index.html:208]: Agreed. I added the word "functionally related", which I hope addresses your concern @iherman.

File: index.html [hunk 6]
```diff
@@ -193,21 +193,32 @@
...
-experimental implementations</a> and a preliminary
-<a href="https://github.com/w3c-ccg/did-test-suite/">test suite</a>
-that will eventually determine whether or not implementations are conformant.
-Readers are advised that Appendix <a href="#current-issues"></a> contains a
-list of concerns and proposed changes that will most likely result in
-alterations to this specification.
+<a href="https://w3c-ccg.github.io/did-method-registry/#did-methods">82
+experimental DID Method specifications</a>, 32 experimental DID Method driver
+implementations, and a Candidate Recommendation <a
+href="https://w3c.github.io/did-test-suite/">test suite</a> that determines
```
[the remark this answers is not in the document]
Comment (msporny) [reply index.html:215]: We've had numbers in here from the very early days. The Status of the Document section is meant to refer to the current status of the document. I do go in and update these numbers from time to time. It is useful to inform readers (and the W3C Director) with an idea of how popular this specification is both wrt. DID methods and DID method implementations.

File: index.html (hunk 3 above)
Comment (msporny) [reply index.html:258]: The names are in chronological order, another ordering that's difficult to argue with. :)

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**